client-sdk/ts-web/core/reflect-go/main.go

```diff
@@ -504,46 +500,32 @@ func write() {
 	}
 }
 
-func main() {
-	visitType(reflect.TypeOf((*beacon.EpochTime)(nil)).Elem())
+// might be nicer to add a function to list these in oasis-core
```

Comment:
Yeah please open up an issue in Oasis Core.

Reply:
👍 filed https://github.com/oasisprotocol/oasis-core/issues/4747